lib/addons/velero/index.ts

```diff
@@ -0,0 +1,259 @@
+import { ClusterAddOn, ClusterInfo } from "../../spi";
+import { Constants } from "..";
+import { Construct } from "@aws-cdk/core";
+import * as s3 from "@aws-cdk/aws-s3";
+import * as iam from "@aws-cdk/aws-iam";
+import { HelmChart, KubernetesManifest, ServiceAccount } from "@aws-cdk/aws-eks";
```

Comment:
`KubernetesManifest` can be removed from import.

Reply:
after addressing Mikhail's comments, it because useful again :)